undertow/src/main/java/org/wildfly/extension/undertow/UndertowSubsystemSchema.java

```diff
@@ -66,10 +66,12 @@ public enum UndertowSubsystemSchema implements PersistentSubsystemSchema<Underto
     VERSION_13_0(13),   // WildFly 27       N.B. There were no schema changes between 12.0 and 13.0!
     VERSION_14_0(14),   // WildFly 28-32
     VERSION_14_0_PREVIEW(14, 0, Stability.PREVIEW),   // WildFly 33-35
-    VERSION_14_0_COMMUNITY(14, 0, Stability.COMMUNITY)   // WildFly 36 - present
+    VERSION_14_0_COMMUNITY(14, 0, Stability.COMMUNITY),   // WildFly 36 - present
+    VERSION_15_0(15),
+    VERSION_15_0_PREVIEW(15, 0, Stability.PREVIEW)
```

Comment:
Are we actually introducing any schema changes between 14.0 and 15.0?
If not, we should not introduce a redundant 15.0 version for Stability.DEFAULT.
If so, we need to introduce a 15.0 for Stability.COMMUNITY as well (as it would include changes introduced in 15.0 Stability.DEFAULT).

Reply:
Ok this sends mixed signals. Ive asked about it and minor versions are supposedly frowned upon and version bump is prefered option.

Reply:
hmm, I might be missing something here. Did not consider having a COMMUNITY stability as base as a valid option.
About your question - I asked about such case - ie there is 14.0_PREVIEW already and there is another RFE with PREVIEW target stability. Answer was that 14.1_PREVIEW is frowned upon and version bump is preffered option.